translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+      "eventMembers": [
+        "**Total Participants**: {players:,}\n\n",
+        "{data}"
+      ]
+    }
+  },
+  "event guild": {
+    "name": "guild",
+    "description": "Megnézi hogyan haladnak egy guild tagjai az eseménnyel.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A keresnivaló guild."
+      }
+    ],
+    "responses": {
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "title": [
+        "A {guild} nevű guild Eseményadatai"
+      ],
+      "guild": [
+        "A **{guild}** nevű guildnak {members} tagja vesz részt az eseményen és együtt átlagolnak {avg:,} {label}-t tagonként. ",
+        "(Total {label}: {total:,})"
+      ],
+      "noMembers": [
+        "A **{guild}** nevű guild-nak nincsenek tagjai az eseményen."
+      ]
+    }
+  },
+  "lookup": {
+    "name": "kikeresés",
+    "description": "Megkeresi, hogy a játékos egy scammer, IRL kereskedő, rattelő vagy makrózó."
+  },
+  "lookup player": {
+    "name": "játékos",
+    "description": "Megkeresi, hogy valaki egy scammer, IRL kereskedő, rattelő vagy makrózó. Megnézi a Discord fiókjukat, ha kötve van hozzá.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A megnézendő játékos."
+      }
+    ],
+    "responses": {
+      "notScammer": [
+        "**{player}** nincs megjelölve scammerként/IRL kereskedőként az adatbázisunkban.\n",
+        "Ez ugyanakkor nem jelenti azt, hogy százszázalékosan nem scammel, mindig légy óvatos, amikor más játékosokkal cserélsz.",
+        "\n\n*Ha ez nem igaz, kérjük jelentsd ezt a játékost `/report` paranccsal.*"
+      ],
+      "isScammer": [
+        "**{player}** scammerként/IRL kereskedőként van megjelölve az adatbázisban."
+      ],
+      "SCAMMER_NAME": [
+        "Bejegyzés #{id} - Scammer"
+      ],
+      "TRADER_NAME": [
+        "Behegyzés #{id} - IRL Kereskedő"
+      ],
+      "RATTER_NAME": [
+        "Bejegyzés #{id} - Rattelő"
+      ],
+      "MACROER_NAME": [
+        "Bejegyzés #{id} - Makrózó"
+      ],
+      "SCAMMER": [
+        "Indok: {reason}"
+      ],
+      "TRADER": [
+        "Indok: {reason}"
+      ],
+      "RATTER": [
+        "Indok: {reason}"
+      ],
+      "MACROER": [
+        "Indok: {reason}"
+      ]
+    }
+  },
+  "lookup user": {
+    "name": "felhasználó",
+    "description": "Megnézi, hogy az adott felhasználó egy scammer/IRL kereskedő/rattelő/makrózó. Megnézi a Minecraft fiókjukat is, ha hozzá van kötve.",
+    "options": [
+      {
+        "name": "felhasználó",
+        "description": "A megnézendő felhasználó."
+      }
+    ],
+    "responses": {
+      "notScammer": [
+        "**{player}** nincs megjelölve scammerként/IRL kereskedőként az adatbázisunkban.\n",
+        "Ez ugyanakkor nem jelenti azt, hogy százszázalékosan nem scammel, mindig légy óvatos, amikor más játékosokkal cserélsz.",
+        "\n\n*Ha ez nem igaz, kérjük jelentsd ezt a játékost `/report` paranccsal.*"
```

Comment:
The command name should be translated
Should also be in backticks for `this effect`